File: beecrowd  1038 - Lanche.py
codigo,qtd = map(int,input().split())
if codigo == 1:
  print('Total: R$ %.2f'%(qtd*4))
elif codigo == 2:
  print('Total: R$ %.2f'%(qtd*4.5))
elif codigo == 3:
  print('Total: R$ %.2f'%(qtd*5))
elif codigo == 4:
  print('Total: R$ %.2f'%(qtd*2))
else:
  print('Total: R$ %.2f'%(qtd*1.5))

File: beecrowd 1038 - Lanche.cpp
#include <iostream>
using namespace std;

int main() {
  int codigo, qtd;
  float total;
  cin >> codigo >> qtd;
  if (codigo == 1){
    total = qtd*4;
  }
  else if (codigo == 2){
    total = qtd*4.5;
  }
  else if (codigo == 3){
    total = qtd*5;
  }
  else if (codigo == 4){
    total = qtd*2;
  }
  else{
    total = qtd*1.5;
  }
  printf("Total: R$ %.2f\n",total);
  
  return 0;

}
